docs/xaml/fundamentals/get-started.md

```diff
@@ -80,6 +87,56 @@ The following example shows a <xref:Microsoft.Maui.Controls.ContentPage> contain
 </ContentPage>
 ```
 
+::: moniker range=">=net-maui-10.0"
+
+## Global XML namespace (preview)
+
+In .NET 10, you can optionally use a global XAML namespace to reduce boilerplate `xmlns:` declarations across your pages.
+
+> [!IMPORTANT]
+> This feature is in preview. To try it, enable preview features in your project file. Behavior and defaults may change.
+
+To opt in, add a `GlobalXmlns.cs` with assembly-level `XmlnsDefinition` attributes that map the global namespace to your CLR namespaces, and reference the `http://schemas.microsoft.com/dotnet/maui/global` namespace at the root of your XAML:
+
+```csharp
+// GlobalXmlns.cs
+using Microsoft.Maui.Controls;
+
+[assembly: XmlnsDefinition("http://schemas.microsoft.com/dotnet/maui/global", "MyApp.Views")]
+[assembly: XmlnsDefinition("http://schemas.microsoft.com/dotnet/maui/global", "MyApp.Controls")]
+```
+
+```xml
+<!-- Root element uses the global namespace -->
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/maui/global"
+                         xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                         x:Class="MyApp.MainPage">
+        <TagView x:DataType="Tag" />
+        <!-- No xmlns prefixes for TagView/Tag when globally mapped -->
+        <!-- Add explicit xmlns prefixes only when disambiguation is needed -->
+    
+</ContentPage>
+```
+
+Optionally, set default prefixes:
+
+```csharp
+using XmlnsPrefixAttribute = Microsoft.Maui.Controls.XmlnsPrefixAttribute;
+
+[assembly: XmlnsPrefix("MyApp.Controls", "controls")]
+```
+
+To enable the most streamlined experience (implicit default namespaces), add:
+
+```xml
+<PropertyGroup>
+    <DefineConstants>$(DefineConstants);MauiAllowImplicitXmlnsDeclaration</DefineConstants>
+    <EnablePreviewFeatures>true</EnablePreviewFeatures>
+</PropertyGroup>
+```
+
+::: moniker-end
+
```

Comment:
those are 2 different features, one of them is preview, and might break tooling. you could make that clearer